src/core.js

```diff
@@ -225,3 +225,8 @@ export function isZoomedOrPanned(chart) {
 
   return false;
 }
+
+export function isZoomingOrPanning(chart) {
```

Comment:
Shouldn't this be called `IsPanningOrDragging`?

Reply:
@LeeLenaleee to me the reasoning sound good, do you have any strong opinions about the name?